mobile_config/src/gateway_service.rs

```diff
@@ -241,18 +240,17 @@ impl mobile_config::Gateway for GatewayService {
 
         let (tx, rx) = tokio::sync::mpsc::channel(100);
 
+        let radios_cache = Arc::clone(&self.tracked_radios_cache);
```

Comment:
`self.tracked_radios_cache` is already an `Arc`, you should be able to just `self.tracked_radios_cache.clone()` no?

Reply:
`Arc::clone(...)` and `self.tracked_radios_cache.clone()` are actually the same under the hood.
When I use `Arc::clone` I just want to emphasize (clarity) in the code that it clones the reference (not the value).


Since it is not crucial for me, I've replaced as you ask